manifests/b/Bultek/SPMInstaller/1.0/Bultek.SPMInstaller.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Bultek.SPMInstaller
+PackageVersion: 1.0

```

```suggestion
PackageVersion: 1.3
```
to follow GitHub Releases